styleguide/source/assets/scss/02-molecules/_jama.scss

```diff
@@ -25,18 +25,23 @@
   }
 
   .ama__link--with-image {
-    img {
-      display: none;
-    }
-
     a {
+      @include breakpoint($bp-small) {
+        width: auto;
+      }
       display: flex;
 
       // For this component, the image is on the right.
       .ama__image__text {
         padding-left: inherit;
         padding-right: 14px;
       }
+      .ama__image-wrap {
+        @include breakpoint($bp-xs max-width) {
+          margin-left: 0;
```

Comment:
@russom-woldezghi ,
Could you rewrite `@include breakpoint($bp-xs max-width)`  so that is defaults to `margin-left: 0` and uses `@include breakpoint($bp-xs)` to set the margin left to auto for other screens? This looks like the only remaining change needed.

Reply:
Thanks @JaeBea. I have updated the `ama__image-wrap` class to the following: 
```
.ama__image-wrap {
 @include breakpoint($bp-xs) {
  margin-left: auto;
 }
}
```